app/assets/stylesheets/home.scss

```diff
@@ -49,6 +49,11 @@
 }
 
 //breakpoints
+@media (max-width: 992px){
+  .home-main-content{
+    width: 345px;
```

Comment:
It's not a good idea to set a fixed width on a specific breakpoint. That will break responsiveness.

The issue was that `Learn Logic Design` was broken in two lines,,
There is 2 solution -
1. Instead of space between the words, use `&nbsp;` tag. If you use this tag this will provide spacing, also the word will never break , so all the three words will be in same line. Although, you need to check whether it works with i18n.
2. Currently those button is in HTML as `<a></a>` tag. You can move them to `button` tag which will also resolve this issue.

I think Solution 2 will be better in this case.

Reply:
ok i will try both solution

Reply:
i tried with 1st approach but it does not look good then i applied 2nd approach 
https://github.com/CircuitVerse/CircuitVerse/assets/116784047/dfd7691f-d3c7-446c-97c7-c93b4c40a599
if any suggestion about this solution ??

Reply:
@tanmoysrt  instead of making the anchor-text to button we can change the padding-right and padding-left of home-main-content container the result is like this 
**before:-**
![image](https://github.com/CircuitVerse/CircuitVerse/assets/116784047/a0501aad-cad8-44b7-99a7-d9ad0a062346)
**after:-**
![image](https://github.com/CircuitVerse/CircuitVerse/assets/116784047/0c355fd5-734d-4cbb-80f0-a362dea2529b)

Reply:
instead of making anchor-text to button we can change padding left and right the result is this 

https://github.com/CircuitVerse/CircuitVerse/assets/116784047/70dace23-5926-4d4d-b427-44d31181240f

Reply:
Push the code, so that I can also check